ui-tests/utils/constants.js

```diff
@@ -6,7 +6,8 @@ const routes = {
   },
   public: {
     home: appBaseUrl,
-    literatureSearch: `${appBaseUrl}/literature`,
+    literatureSearch: `${appBaseUrl}/literature/customSearch`,
+    getLiteratureDetailForRecord1: () => 'http://localhost:8080/literature/1',
```

Comment:
I've also tried replacing this with:
`teratureDetailForRecord1: () => `${appBaseUrl}/literature/1`,`

Reply:
* locahost:8080 should be hardcoded here
* The reason why I suggested you to use a function is because you can parameterize the record id but now having a function in this case is pointless. 

You should do one of the below, depending on weather you need to parameterize `recordId` or not.

```js
getLiteratureDetailForRecordId: (recordId) => `${appBaseUrl}/literature/${recordId}`
```

or 

```js
literatureDetailForRecord1: `${appBaseUrl}/literature/1`
```

Reply:
Okay, for now I'll use the latter approach (since the mock data I have is also only for a record with control number 1)